rust-toolchain.toml

```diff
@@ -1,3 +1,2 @@
 [toolchain]
-channel = "nightly"
```

Comment:
at one point we were dependent on `nightly` for one of our dependencies; is this no longer the case?